pytest tests/models/test_dagcode.py::TestDagCode::test_sync_to_db # Error
pytest tests/models/test_dagcode.py::TestDagCode::test_bulk_sync_to_db
pytest tests/models/test_dagcode.py::TestDagCode::test_bulk_sync_to_db_half_files

# 혹은 test_dagcode.py 전체가 관련된 테스트일수도!